### DISCUSSION TITLE
[FR] Add API endpoint to get developer plugins

### DISCUSSION THREAD

**Discussion by khalwat** [craftcms/console#92] [Craft Console]
It'd be nice if CraftNet had an API endpoint to query a plugin data the way it has one for licenses.

I'm interested in it primarily so I can _get_ all plugins, _get_ information about them, etc. But it also could potentially be used to _set_ information as well, which could come in handy.

**Comment by BenParizek** [comment]
It would be nice to be able to query for a single plugin, or group the results of the all plugin query to be grouped by plugin or customer.

**Comment by khalwat** [comment]
Yeah generalizing the API to allow for more explicit queries would be nice... or a GraphQL endpoint so we can do whatevs.
